changelog/unreleased/features/1611.md

```diff
@@ -0,0 +1,4 @@
+The new options `vast export --timeout=<duration>` and `vast export
+--error-on-timeout` allow for setting a timeout for VAST queries. If
+`error-on-timeout` is enabled, cancelled exports result in a non-zero
+exit code.
```

Comment:
If the timeout gets hit then the exit code should ALWAYS be non-zero. The option `error-on-timeout` makes no sense to me.

Otherwise the functionality seems to work and the code looks ok.

Reply:
I changed it accordingly, and also fixed the symlink to `vast.yaml.example` from `vast/integration/vast.yaml.example`, which we seemed to have accidentally converted into a hard copy instead.